deps/build.jl

```diff
@@ -22,4 +29,4 @@ if (!ispath("../usr/lib"))
     run(`mkdir ../usr/lib`)
 end
 
-run(`mv libwrapclang.$(Libdl.dlext) ../usr/lib`)
+run(`mv libwrapclang.$(dlext) ../usr/lib`)
```

Comment:
Also I don't see the point of this change either.
`Clang.jl` requires julia 0.4 and AFAICT `Libdl.dlext` is available ever since `Libdl` is introduced. It's not like you are removing all direct references to `Libdl` (not even in this file) either.

Also AFAICT, the version that introduces Libdl is `0.4.0-dev+3840` (Compat.jl uses `0.4.0-dev+3844`) and not the version you specify here.